src/general/StageHUDBase.cs

```diff
@@ -903,33 +903,33 @@ protected Color GetCompoundDensityCategoryColor(float amount)
     /// <summary>
     ///   Updates the compound bars with the correct values.
     /// </summary>
-    protected virtual void UpdateCompoundBars()
+    protected virtual void UpdateCompoundBars(float delta)
     {
         var compounds = GetPlayerStorage();
 
         glucoseBar.MaxValue = compounds.GetCapacityForCompound(glucose);
-        glucoseBar.Value = compounds.GetCompoundAmount(glucose);
+        GUICommon.SmoothlyUpdateBar(glucoseBar, compounds.GetCompoundAmount(glucose), delta);
```

Comment:
I really hope these don't cause similar bar stuck bugs as we've seen with the ATP and health bars...

Reply:
This static method was made to fix that very problem.